src/main/java/net/mcreator/ui/component/SearchableComboBox.java

```diff
@@ -137,6 +141,10 @@ private void clearSearch() {
 		comboFilter();
 	}
 
+	public boolean isSelectingOrFiltering() {

```

Comment:
SearchableComboBox is selecting while searching, so this will likely not work well and may miss some code updates

Reply:
You mean the only `setSelectedItem()` call in there? `filtering` is set to false before that happens.

Reply:
No. Removing currently selected element can select other element.

Unrelated, please see my comment on feasibility of this PR

Reply:
This is done via utility class, so we can't do much during that.

Reply:
This is done via utility class, so we can't do much during that.

Reply:
`ComboBoxUtil`. More precisely, `ComboBoxUtil.updateComboBoxContents()`.